odl/diagnostics/operator.py

```diff
@@ -167,18 +167,18 @@ def _adjoint_of_adjoint(self):
 
         with FailCounter('error = ||Ax - (A^T)^T x|| / '
                          '||A|| ||x||') as counter:
-            for [n_x, x] in vector_examples(self.operator.domain):
-                A_result = self.operator(x)
-                ATT_result = self.operator.adjoint.adjoint(x)
+            for [name_x, x] in vector_examples(self.operator.domain):
+                opx = self.operator(x)
+                transp_transp_x = self.operator.adjoint.adjoint(x)
```

Comment:
we should use `adj`/`adjoint` consistently, helps users if we stick to one.

`op_adj_adj_x`